list_state_counterparts.rq

```diff
@@ -6,7 +6,9 @@ PREFIX owl: <http://www.w3.org/2002/07/owl#>
 PREFIX dc: <http://purl.org/dc/elements/1.1/>
 PREFIX StandardState: <http://purl.obolibrary.org/obo/CDAO_0000045>
 PREFIX bd: <http://www.bigdata.com/rdf/search#>
-# replace the value of the following prefix with the character URI of interest
+# Replace the value of the following prefix with the character URI of interest
+# These URIs are not persistent across KB updates; this example will return no 
+# results unless you provide a valid character URI.
```

Comment:
Same as for #7 -- is it feasible to simply sample (first or last, or something) from character URIs in the database?

Again, this would only make sense if the assumption holds that there are no character URIs in the database that wouldn't produce a row in this query's result.